src/globals.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * globals.ts hold some globals used throughout the extension
+ */
+
+// true for running tests, false during regular runtime
+declare var isTesting: boolean;
```

Comment:
which object, are you saying I should make a globals class that is exported?

export class globals {
  public isTesting: boolean = false;
}